lunatrace/bsl/hasura/migrations/lunatrace/1664218225933_release_analysis_result/up.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TYPE "finding_state" AS ENUM (
```

Comment:
I haven't been able to figure out how to update an enum inside of a transaction in psql. Additionally, because of how hasura is with these types, I think we should stick to doing table enums.

Reply:
@factoidforrest wdyt about this? We previously decided to use native enums for consistency but I think it may be worth cutting over. We can make a migration to switch everything to table enums.

Reply:
I would agree about renaming it because "state" is redundant.

Yes, there will be more values. We're eventually hoping to have all the values from the [VEX part of CycloneDX](https://github.com/CycloneDX/specification/blob/master/schema/bom-1.4.proto#L622-L643) and more, so that's why we would want to edit it later.

I think that makes a pretty good case for table enum in this case.